lisa/tests/scheduler/load_tracking.py

```diff
@@ -53,13 +55,64 @@ class LoadTrackingHelpers:
     """
     Common bunch of helpers for load tracking tests.
     """
-    @staticmethod
-    def get_max_capa_cpu(te):
+
+    MAX_RTAPP_CALIB_DEVIATION = 3/100
```

Comment:
I gave this a try on my HiKey960 and CPU0 is still picked up. FYI the calibration I get is:
```python
{0: 308, 1: 302, 2: 302, 3: 303, 4: 155, 5: 155, 6: 155, 7: 155}
```

So looking at this, maybe it would be simpler to get rid of it and ditch the load tests. I've never really trusted them anyway - a tiny (timewise) task with a huge priority could mess the whole test up.

Reply:
For now I'd rather not ditch the test until it's proven to be unstable (i.e. going through one round of integration). That said we might want to turn that threshold into an absolute one instead of a percentage. On my experiments, I got no issues with load testing on hikey 960 at all frequencies on all capacity classes.

Reply:
They all passed for me, but I just don't really see a value in asserting the load values.

It would make more sense to have something like run the same task but change its niceness and assert that it did change the load. Or voluntarily create an imbalanced scenario and make sure there is a significant load difference. In this case (load for a rq with a single test task) it's bound to be more painful than helpful.

I'm not saying this is to rip out now, but my 2cents for "the future".

Reply:
> I just don't really see a value in asserting the load values.

It could break independently from util signal, so still worth testing if that's easy. In our case, it is easy (unless proven to be really flaky in practice), so we might as well do it.